AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -86,11 +86,13 @@ def deferred_fog(property: str = 'name') -> str:
           - 'requires' a list of component names as strings required by this component.
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
         :param property: From the last element of the property tree path. Default 'name' for component name string.
+         - 'Enable Deferred Fog' Toggle active state of the component True/False
```

Comment:
this should go above the line :param property:

Reply:
Fixed. Not sure how I missed that. Thanks.